### PR TITLE
CT-858 udpate table delete rows use JSON

### PR DESCRIPTION
**Pull request by martinjunger**
Jira: CT-858
Connection: https://github.com/keboola/connection/pull/4210

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/CommonPart1/DeleteRowsTest.php [hunk 1]
```diff
@@ -91,13 +87,13 @@ public function tableDeleteRowsByFiltersData(): array
             ],
             'since yesterday - timestamp' => [
                 [
-                    'changedSince' => $yesterday->getTimestamp(),
+                    'changedSince' => (string) $yesterday->getTimestamp(),
```
Comment (jirkasemmler) [inline tests/Backend/CommonPart1/DeleteRowsTest.php:90]: tohle udelat nemuzeme. Test to sice fixuje, ale je to BC break. Prepnuti `apiDeleteParams` -> `apiDeleteParamsJson` zpusobi, ze ten metoda `deleteTableRows` nebude fungovat (bude strilet 400), kdyz tam nekdo posle klientem timestamp jako int. Takze navrhuju, aby to klient v metode `deleteTableRows` checknout a zkonvertovat na string. Jinak to bude BC break.

Nebo to pak vydat jako minor/major a informovat o tom, aby si minimalne sluzby to osetrily. 

Nebo to vyresit v connection v RO

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
musime vyresit ten timestamp

File: tests/Backend/CommonPart1/DeleteRowsTest.php (hunk 1 above)
Comment (martinjunger) [reply tests/Backend/CommonPart1/DeleteRowsTest.php:90]: hmm, dobrej point. Sel jsem primarne podle dokumentace, tam je klasicky `string`.
Takze to upravim v Connection, aby to prijimalo int|string a bude to osetreny...

Comment (martinjunger) [reply tests/Backend/CommonPart1/DeleteRowsTest.php]: fixnuto

**Review by jirkasemmler — APPROVED** [review]

**Pull request merged**